### PR TITLE
9: add changelog entry for setup.cfg configuration

### PR DESCRIPTION
**Pull request by gotmax23**
Relates: https://github.com/ansible-community/antsibull/pull/530

### CODE REVIEW

File: 9/changelog.yaml [hunk 1]
```diff
@@ -2,6 +2,13 @@ ancestor: 8.0.0
 releases:
   9.0.0a1:
     changes:
+      minor_changes:
```
Comment (felixfontein) [inline 9/changelog.yaml:5]: I wonder whether this should be moved to
```suggestion
      major_changes:
```
so it shows up in the porting guide? On the other hand, this should not really affect anyone directly. WDYT?

Comment (gotmax23) [reply 9/changelog.yaml:5]: Yeah, it shouldn't really affect anyone except maybe Linux distributions that patch the setup.py in [some way](https://src.fedoraproject.org/rpms/ansible/blob/0321735d7fc291639db0b22572e0d5bd812b7d97/f/ansible.spec#_91). I'm fine with adding it to the Porting Guide, but I don't think it *needs* to be there.

Comment (felixfontein) [reply 9/changelog.yaml:5]: In that case, let's keep it as-is.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
We should announce this (again) in #ansible-packaging. Besides that, I think it's good to go!

**Pull request merged**